README.md

```diff
@@ -568,8 +569,8 @@ A curated list of awesome C/C++ frameworks, libraries, resources, and shiny thin
 
 * [8cc](https://github.com/rui314/8cc) - A Small C Compiler.
 * [c](https://github.com/ryanmjacobs/c) - Compile and execute C "scripts" in one go! [MIT]
-* [Clang](http://clang.llvm.org/) - A C compiler for LLVM. Supports C11. Developed by Apple. [NCSA]
-* [GCC](https://gcc.gnu.org/) - GNU Compiler Collection. Supports C11 and OpenMP. [GNU GPL3]
+* [Clang](http://clang.llvm.org/) - A C compiler for LLVM. Supports C++11/14/1y C11. Developed by LLVM Team. [NCSA]
```

Comment:
Three points here:
* C++1y was the working title for C++14, whereas the upcoming version ("C++17", if that's what you wanted to include) is C++1z. If that's not what you meant, I would suggest just leaving it out, in the same way that you don't list C++03 support either because it's implied.
* It should be "C++11/14/1z `and` C" (note the "and"). Please add.
* "Developed by LLVM team" is technically true but less helpful than to know that Apple is its biggest supporter. Maybe instead of "Developed by Apple", it could be "Backed by Apple"?